app/src/main/java/com/example/una/utils/ChallengeViewsUtil.java

```diff
@@ -38,10 +38,9 @@
 
 public class ChallengeViewsUtil {
     public static final String TAG = "ChallengeViewsUtil";
-    static FirestoreClient client = new FirestoreClient();
 
     // set text view with donor-recipient information
-    public static void setTvOwnerRecipientInfo(TextView tvChallengeOwnerRecipientInfo, Challenge challenge) {
+    public static void setTvOwnerRecipientInfo(FirestoreClient client, TextView tvChallengeOwnerRecipientInfo, Challenge challenge) {
```

Comment:
is the `FirestoreClient` used in this file?

Reply:
Yeah, I had to make some changes for the FirestoreClient constructor to take in a context, so I had to add it as a parameter for a few of the methods in this class